actioncable/CHANGELOG.md

```diff
@@ -1,2 +1,13 @@
+*   Compress large payloads in all subscription adapters
+
+    This allows to broadcast payloads larger than 8000 bytes by compressing them with ActiveSupport::Gzip with PostgreSQL
```

Comment:
Think this needs a rewording.

Reply:
Also, maybe give some indication of how big the payload can be now. Before we blew at 8k. What should you expect to be the limit with compression now?

Reply:
Maybe double? I guess it depends a lot on the string. Also the limit really only gives problems in PostgreSQL.

Reply:
To me it's about more than the limit. It's simply more efficient! We use compression for everything else that goes over the wire. So would make a combined point.

Reply:
Good point. What do you think now?